README_zh.md

```diff
@@ -225,7 +255,6 @@ daemonset.apps/edgemesh-agent created
 ```
 
 
-
 #### 测试样例
 
 **HTTP协议**
```

Comment:
`kubectl apply -f example/hostname.yaml` can be modified to `kubectl apply -f examples/hostname.yaml`

Reply:
fix